src/components/Chat/BottomSheetGif.js

```diff
@@ -27,18 +28,10 @@ const BottomSheetGif = React.forwardRef((props, ref) => {
   const [list, setList] = React.useState([]);
   const [isLoading, setIsLoading] = React.useState(true);
 
-  const handleChangeText = (text) => {
-    setSearchText(text);
-  };
-
-  const handleOnClearText = () => {
-    setSearchText('');
-  };
-
-  const onLoad = async () => {
+  const onLoad = async (search) => {
     setIsLoading(true);
     try {
-      const result = await getGifFeatured();
+      const result = await getGifFeatured(search);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The modification of the `onLoad` function to accept a `search` parameter is a good change. It allows the function to be more flexible and reusable for different search queries. However, there's no error handling if the `getGifFeatured` function fails. Consider adding a catch block to handle potential errors.
```diff
  const onLoad = async (search) => {
     setIsLoading(true);
     try {
      const result = await getGifFeatured(search);
+    } catch (error) {
+      console.error(error);
+      // Handle the error appropriately
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->